rts/Net/AutohostInterface.cpp

```diff
@@ -24,73 +24,127 @@ LOG_REGISTER_SECTION_GLOBAL(LOG_SECTION_AUTOHOST_INTERFACE)
 namespace {
 
 /**
- * @enum EVENT Which events can be sent to the autohost
- *   (in brackets: parameters, where uchar means unsigned char and "string"
- *   means plain ascii text)
+ * @enum EVENT Which events can be sent to the autohost.
+ *
+ * Each packet from engine to autohost interface starts with a byte that
+ * indicated the type of the message. Below in brackets we describe the data
+ * associated with each message.
```

Comment:
Could document how `char[]` is delimited (presumably `\0` but good to be explicit)

Reply:
They are not delimited in any way, added note.